src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateEvaluator.cs

```diff
@@ -353,6 +367,24 @@ private TemplateContext CreateContext(DictionaryContextData contextData)
                 }
             }
 
+            // Add functions
```

Comment:
may need to add functions too b/c of early display name scenario... look into this more tomorrow...

Reply:
i wonder if templatereader/templateevaluator are checking for this during avail/legal check